src/components/Chat/ChatGutter.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+
+import {
+  UIComponent,
+  childrenExist,
+  UIComponentProps,
+  ContentComponentProps,
+  ChildrenComponentProps,
+  commonPropTypes,
+  createShorthandFactory,
+} from '../../lib'
+import { Extendable, ShorthandValue } from '../../../types/utils'
+
+export interface ChatGutterProps
```

Comment:
Still not sure why we decided to add this component. It is identical to the `Slot` component which was previously used here. The `Slot` can still be used as a shorthand in the `ChatItem`. Maybe I am missing something?

Reply:
It is just another component that doesn't bring any value in my opinion.

Reply:
1. consistency between having a slot and subcomponent? @levithomason  maybe we should revisit this since @mnajdova has a point: `Button` has `icon` slot but there is not `Button.Icon`

2. custom styling

Reply:
removed `Chat.Gutter`